test/unit/lib/classes/ConfigSchemaHandler/normalizeAjvErrors.test.js

```diff
@@ -283,10 +283,10 @@ describe('#normalizeAjvErrors', () => {
       () =>
         expect(
           errors.some(error => {
-            if (error.dataPath !== '.provider.deploymentBucket.maxPreviousDeploymentArtifacts') {
+            if (error.dataPath !== '/provider') {
               return false;
             }
-            if (error.keyword !== 'type') return false;
+            if (error.keyword !== 'anyOf') return false;
```

Comment:
In AJV v6, our error normalizer stripped the `anyOf` error for `.provider` and was surfacing only the meaningful `.provider.deploymentBucket.maxPreviousDeploymentArtifacts` error, and this is what this test case confirms.

Technically it should remain same (just with path separators replaced).

Is the case here that AJV no longer reports the error for `/provider/deploymentBucket/maxPreviousDeploymentArtifact` property?

Reply:
@medikoo my mistake there. `resolveDataPathSize` util was still used to compute path depth. This util is no longer required as all properties separator are `/` instead of `.` or `[`/`]`. I updated the PR accordingly and this test is no longer changed